app/assets/stylesheets/custom.scss

```diff
@@ -6,16 +6,38 @@
 //header logo
 #logo {
   float: left;
-  margin-right: 10px;
+  margin-left: 25px;
   font-size: 1.8em;
+  font-family: Arial;
   color: #FFD600;
   text-transform: capitalize;
-  letter-spacing: -1px;
-  padding-top: 5px;
+  letter-spacing: 0px;
+  padding-top: 8px;
   font-weight: 900 ;
   &:hover {
     color:deepskyblue;
     text-decoration: none;
   }
 }
 
+#login{
+  float: right;
+  margin-top: 10px;
+  margin-right: 18px;
+  font-size: 1.2em;
+  font-family: Arial ;
+}
+
+// header drop-down items
+div.dropdown-item{
+  font-family: Arial;
+
+}
+span.user-name{
+  font-family: Arial;
+}
+
+//error messages
+div.alert{
```

Comment:
why? TOO global change. Remove this and rework UI. We will come back to this question later.